<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WDSPRxSquelchDialog</class>
 <widget class="QDialog" name="WDSPRxSquelchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Squelch Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="voiceSquelchGroup">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox { 
     border: 1px solid gray; 
 } </string>
     </property>
     <property name="title">
      <string>Voice Squelch (SSB)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="ssqlTauMuteLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Tau mute (s)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="ssqlTauMute">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>70</width>
        <height>29</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add yaw offset to help with aligning images on the map.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>2.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="ssqlTauUnmuteLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Tau unmute (s)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="ssqlTauUnmute">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>70</width>
        <height>29</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add yaw offset to help with aligning images on the map.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="amSquelchGroup">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox { 
     border: 1px solid gray; 
 } </string>
     </property>
     <property name="title">
      <string>AM Squelch (AM, SAM, CW)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="amsqMaxTailLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Tail (s)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="amsqMaxTail">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>70</width>
        <height>29</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add yaw offset to help with aligning images on the map.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>3.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.500000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fmSquelchGroup">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox { 
     border: 1px solid gray; 
 } </string>
     </property>
     <property name="title">
      <string>FM Squelch (FM)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WDSPRxSquelchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WDSPRxSquelchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
